packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -219,6 +219,9 @@ export class DefaultWeb3Handler implements Web3Handler, FullnodeHandler {
         this.assertParameters(params, [])
         response = await this.chainId()
         break
+      case Web3RpcMethods.traceTransaction:
+        response = await this.traceTransaction(params[0], params[1])
+        break
```

Comment:
Should this go in `test-web3-rpc-handler.ts`? I suppose we won't be using the main one anymore, but still.